src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaData.java

```diff
@@ -74,4 +74,11 @@ public String toString() {
                 "repositoryPath='" + repositoryPath + '\'' +
                 '}';
     }
+
+    @Override
+    public int compareTo(@NotNull SlingInitialContentBundleEntryMetaData o) {
+        return this.repositoryPath.compareTo(o.repositoryPath);
```

Comment:
i am not convinced that string comparison is sensible for paths. what is the goal here?